source/Octopus.Tentacle.Tests.Integration/Support/ClientAndTentacle.cs

```diff
@@ -53,10 +54,16 @@ public async ValueTask DisposeAsync()
         {
             SafelyMoveTentacleLogFileToSharedLocation();
 
-            logger.Information("****** ****** ****** ****** ****** ****** ******");
-            logger.Information("****** CLIENT AND TENTACLE DISPOSE CALLED  *****");
-            logger.Information("*     Subsequent errors should be ignored      *");
-            logger.Information("****** ****** ****** ****** ****** ****** ******");
+            var banner = new StringBuilder();
```

Comment:
Slightly easier to read log 
![image](https://github.com/OctopusDeploy/OctopusTentacle/assets/86938706/a2404e03-c1b4-4c12-a460-6b6650d2d5a1)